### PR TITLE
Updated FedRAMP Baselines

### PR DESCRIPTION
**Pull request by brianrufgsa**
# Committer Notes

Updated FedRAMP baselines as OSCAL profiles and resolved profiles. 
These use the revised profile schema with the new alter/add/@id-ref syntax

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: docs/content/contribute/devlunch.md [hunk 1]
```diff
@@ -3,9 +3,17 @@ title: Lunch with the Devs
 description: Contact Us
 ---
 
-{{% usa-intro %}}To increase communication with the OSCAL community, the NIST OSCAL team is hosting a one-hour teleconference every two weeks starting on Thursday, December 5th 2019 @ noon EST.{{% /usa-intro %}}
+{{% usa-intro %}}To increase communication with the OSCAL community, the NIST OSCAL team is hosting a one-hour teleconference every two weeks starting on Thursday, December 5th 2019 @ noon EST.{{% /usa-intro %}}Global
```
Comment (bradh) [inline docs/content/contribute/devlunch.md]: This change probably doesn't belong in this PR

Comment (david-waltermire-nist) [reply docs/content/contribute/devlunch.md]: Agreed. We need to do a better job of keeping PRs more focused on set of related changes.

**Comment by david-waltermire-nist** [comment]
@brianrufgsa Can you rebase this against master?

To do this you can run the command:
```
git pull -r upstream master
```

File: src/metaschema/oscal_profile_metaschema.xml [hunk 2]
```diff
@@ -392,15 +411,5 @@
...
-   <define-flag as-type="NCName" name="position">
-      <formal-name>Position</formal-name>
-      <description>Where to add the new content with respect to the targeted element (beside it or inside it)</description>
-      <allowed-values><!-- TODO: define these -->
-         <enum value="before"/>
-         <enum value="after"/>
-         <enum value="starting"/>
-         <enum value="ending"/>
-      </allowed-values>
-   </define-flag>
```
Comment (david-waltermire-nist) [inline src/metaschema/oscal_profile_metaschema.xml]: @brianrufgsa Can you explain this change?

File: xml/convert/oscal_catalog_json-to-xml-converter.xsl [hunk 3]
```diff
@@ -1289,6 +1289,7 @@
          <xsl:apply-templates mode="#current" select="*[@key=('title')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('param', 'parameters')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('prop', 'properties')]"/>
+         <xsl:apply-templates mode="#current" select="*[@key=('annotation', 'annotations')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('link', 'links')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('part', 'parts')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('control', 'controls')]"/>
```
Comment (david-waltermire-nist) [inline xml/convert/oscal_catalog_json-to-xml-converter.xsl]: @brianrufgsa You shouldn't commit any changes against files in the xml, json, or content directories, since these files are generated by the CI/CD. Can you revert these files?

File: src/metaschema/oscal_profile_metaschema.xml (hunk 2 above)
Comment (brianrufgsa) [reply src/metaschema/oscal_profile_metaschema.xml]: @david-waltermire-nist I did all that, and when I prepared the pull request it only showed my eight files. I am not sure what is happening. It was only supposed to be the four FedRAMP baselines and resolved profiles.
